src/backend/joanie/tests/core/test_api_certificate.py

```diff
@@ -15,16 +15,17 @@
 class CertificateApiTest(BaseAPITestCase):
     """Certificate API test case."""
 
+    maxDiff = None
```

Comment:
Debug residue ?

Reply:
No, it's used with assertDictEqual. This allows to display clearly the difference between the expected and the actual dict.

Reply:
Ok let's keep it